packages/flutter_tools/lib/src/base/utils.dart

```diff
@@ -8,6 +8,12 @@ import 'dart:io';
 import 'package:crypto/crypto.dart';
 import 'package:path/path.dart' as path;
 
+bool isRunningOnTravis() {
```

Comment:
isRunningOnCI maybe?

Also, this could be a getter, instead of a method. Your call.